deploy/addons/addon-manager.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Ah, didn't notice the volume mounting difference between the two.
Then you may move this to `deploy/iso/addon-manager.yaml` or `deploy/iso/static-pods/addon-manager.yaml`

Also, please move `deploy/static-pods/addon-manager.yaml` to `deploy/docker/static-pods/addon-manager.yaml`